CHANGELOG.adoc

```diff
@@ -5,7 +5,7 @@
 === Breaking changes
 
 - https://github.com/eclipse-syson/syson/issues/298[#298] [syson] Add standard libraries and new models now have a root Namespace to conform with KerML/SysML specifications.
-
+- https://github.com/eclipse-syson/syson/issues/349[#349] [core] Switch to sirius-web domain driven design architecture.
```

Comment:
There is no `[core]` topic in syson's changelog. Could you please change to `[syson]`?

Reply:
Done